test/functional/requirements.txt

```diff
@@ -1 +1,2 @@
 attotime>=0.2.0
+schema==0.7.2
```

Comment:
Does it have to be equal or can it be ">="?

Reply:
I think it could be. I just pinned it to this specific version, because that's the one I tested with.